src/lib/stringIndices.js

```diff
@@ -1,16 +1,19 @@
-  let stringIndices = (userString, userNumber) => {
+
+let stringIndices = (userString, userNumber) => {
     let newArr = []
     let separate = ""
+    let emptyString = " "
+
+    if(userNumber > newArr || userNumber < 0){
+      return emptyString
```

Comment:
I would just return the empty string directly:
```js
return ''
```
This prevents me from having to look up the value of `emptyString`.  Also, you're returning a space `' '`, not the empty string `''`.